Source/stores.cpp

```diff
@@ -201,11 +202,13 @@ void AddSTextVal(int y, int val)
 	stext[y]._sval = val;
 }
 
-void AddSText(int x, int y, const char *str, UiFlags flags, bool sel)
+void AddSText(int x, int y, string_view text, UiFlags flags, bool sel)
 {
 	stext[y]._sx = x;
 	stext[y]._syoff = 0;
-	strcpy(stext[y]._sstr, str);
+	int len = std::min<int>(text.length(), 127);
+	strncpy(stext[y]._sstr, text.data(), len);
+	stext[y]._sstr[len] = '\0';

```

Comment:
```suggestion
	CopyUtf8(stext[y]._sstr, text, sizeof(stext[y]._sstr));
```

(after including `utils/utf8.hpp`)

Reply:
Thanks. I had a feeling I forgot this in a couple of places, but was to tired to properly go over it again :)